types/artifacts.d.ts

```diff
@@ -169,6 +171,14 @@ declare global {
         params: {name: string; value: string}[];
       }
 
+      /** @see https://developer.mozilla.org/en-US/docs/Web/HTML/Element/link#Attributes */
+      export interface LinkElement {
+        rel?: string
+        href?: string
+        as?: string
+        crossorigin?: string
```

Comment:
shouldn't these all be required since the browser will normalize to at least an empty string? Or we could have the gatherer explicitly drop `''` in favor of `undefined`.

And from the spec/a quick test, `crossorigin` can be `null | 'anonymous' | 'use-credentials'` (the browser corrects anything else to `'anonymous')

Reply:
yeah, SGTM I'll change to match spec